### PR TITLE
ERR: better error reporting for failing parsing in timedelta/timeseries #10720

### LINKED ISSUE #10720: DOC/ERR: better error message on unsuccessful datetime parsing
exceptions that are raised on unsuccessful datetime/timedelta parsing should add this:

``you can coerce to NaT by passing errors='coerce'``

comment at the end: https://github.com/pydata/pandas/pull/10674

### PR DESCRIPTION
**Pull request by springcoil**
closes #10720

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
Why did you close this one?

**Comment by jorisvandenbossche** [comment]
Some advice:

- always work on a branch and not in master for a PR (so this one is better than #10884, I can reopen this one if you want)
- always update with the latest upstream before starting a new branch (you have still some old commits of a previous pr):
  ```
git checkout master
git fetch upstream
git merge upstream/master --ff-only
git checkout -b new_branch
  ```

**Comment by springcoil** [comment]
Ohh i closed the other one because i wanted to rebase. Thanks for the tip

On Saturday, 22 August 2015, Joris Van den Bossche <notifications@github.com>
wrote:

> Some advice:
>
>    - always work on a branch and not in master for a PR (so this one is
>    better than #10884 <https://github.com/pydata/pandas/pull/10884>, I
>    can reopen this one if you want)
>    - always update with the latest upstream before starting a new branch
>    (you have still some old commits of a previous pr): git checkout
>    master git fetch upstream git merge upstream/master --ff-only git checkout
>    -b new_branch
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/10883#issuecomment-133674560>.
>


-- 
Peadar Coyle
Skype: springcoilarch
www.twitter.com/springcoil
peadarcoyle.wordpress.com

**Comment by springcoil** [comment]
Can you reopen this one instead and close the other one?

**Comment by springcoil** [comment]
Ok - so that attempt passed, any got any feedback about whether it's ready to be merged or does it need some more work?

File: pandas/tseries/tests/test_timeseries.py [hunk 1]
```diff
@@ -923,8 +923,8 @@ def test_to_datetime_with_apply(self):
         assert_series_equal(result, expected)
 
         td = pd.Series(['May 04', 'Jun 02', ''], index=[1,2,3])
-        self.assertRaises(ValueError, lambda : pd.to_datetime(td,format='%b %y', errors='raise'))
-        self.assertRaises(ValueError, lambda : td.apply(pd.to_datetime, format='%b %y', errors='raise'))
+        self.assertRaises(ValueError, 'Unknown string format. You can coerce errors to NaT by passing coerce', lambda : pd.to_datetime(td,format='%b %y', errors='raise'))
```
Comment (jreback) [inline pandas/tseries/tests/test_timeseries.py]: his is not doing anything

use assertRaisesRegExp

**Comment by springcoil** [comment]
Seems ready

**Comment by jreback** [comment]
@springcoil this doesn't do anything. as you didn't actually change any code except for the tests. So this fails. And it still is not rebased on master.

In [1]: td = pd.Series(['May 04', 'Jun 02', ''], index=[1,2,3])

In [3]: pd.to_datetime(td,format='%b %y', errors='raise') 
ValueError: time data '' does not match format '%b %y' (match)
```

**Comment by springcoil** [comment]
Ok will fix that

On Saturday, 22 August 2015, Jeff Reback <notifications@github.com> wrote:

> @springcoil <https://github.com/springcoil> this doesn't do anything. as
> you didn't actually change any code except for the tests. So this fails.
> And it still is not rebased on master.
>
> In [1]: td = pd.Series(['May 04', 'Jun 02', ''], index=[1,2,3])
>
> In [3]: pd.to_datetime(td,format='%b %y', errors='raise')
> ValueError: time data '' does not match format '%b %y' (match)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/10883#issuecomment-133755403>.
>


-- 
Peadar Coyle
Skype: springcoilarch
www.twitter.com/springcoil
peadarcoyle.wordpress.com

**Comment by springcoil** [comment]
I'm going to close this for now.

**Pull request closed without merging**